spec/features/facets_spec.rb

```diff
@@ -67,6 +67,26 @@
     expect(page).to have_css('#facet-format', visible: true) # assert that it didn't re-collapse
   end
 
+  describe 'Facet heading button focus with Firefox' do
+    before do
+      Capybara.current_driver = :selenium_headless
```

Comment:
I think you can omit this and use `js: true` as argments to the `describe` (same with the `after`)

Reply:
Kinda goes with my other comment, basically I'm trying to use FF not Chrome, and headless Chrome is what is called with `js: true` I believe.